pkg/docker/pusher.go

```diff
@@ -140,102 +140,15 @@ func (p dockerPusher) push(ctx context.Context, desc ocispec.Descriptor, src res
 		req = p.request(host, http.MethodPut, putPath...)
 		req.header.Add("Content-Type", desc.MediaType)
 	} else {
-		// Start upload request
```

Comment:
I completely understand, why you restructured this code. The change looks ok.
But this code has been copied from  `github.com/containerd/containerd remotes/docker` (see README) to fix the retry behavior.
I would like to keep it as close as possible to the original to make it easier to handle changes in the original version.

Reply:
I see.

This comment, at the top of the file, told me it's "original content":
```
// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Open Component Model contributors.
//
// SPDX-License-Identifier: Apache-2.0
```

I'll revert the changes and add this directory to the ignore list (with a comment why ignored).

Reply:
done.

Actions taken:
- reverted all changed under `pkg/docker`
- disabled all checks for `pkg/docker`